.vscode/launch.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Probably okay for this personal project, but I'd recommend adding `.vscode` to your `.gitignore`. We don't generally check in editor configs, as other developers may have different setups, tho I can see you've got specific instructions in your README about running the application from VSCode, so maybe a good argument for keeping this in.

Reply:
I get it, I'm just trying to use this project to track my personal rails vscode set up.